torchtune/modules/transforms/tokenizers/_utils.py

```diff
@@ -12,7 +12,7 @@
 from torchtune.data._utils import truncate
 from torchtune.utils import get_logger
 
-_log: logging.Logger = get_logger()
+log: logging.Logger = get_logger()
```

Comment:
Can also probably revert the changes to this file and transformer.py now to match main